tests/manager/searchCustomer/managerCanSearchCustomerByPostalCode.spec.js

```
@@ -17,14 +24,34 @@ test.beforeEach(async ({ page }) => {
   firstName = faker.person.firstName();
   lastName = faker.person.lastName();
   postalCode = faker.location.zipCode();
+
+  await addCustomerPage.openAddCustomerPage();
+  await addCustomerPage.waitForAddCustomerPage();
+  await addCustomerPage.fillFirstNameField(firstName)
+  await addCustomerPage.fillLastNameField(lastName)
+  await addCustomerPage.fillPostCodeField(postalCode)
+  await addCustomerPage.clickAddCustomerButton()
+  await page.reload()
```

Reconsider relying on `page.reload()` after clicking Add Customer. Reloading to dismiss the success popup is brittle and can mask failures. Prefer handling the alert (if it's a dialog) with `page.on('dialog', d => d.accept())` or waiting for a specific UI change (e.g., a success message disappears or the Customers button becomes enabled). This will make setup deterministic and less prone to timing issues.